ee/dtr/admin/install/index.md

```diff
@@ -131,13 +130,12 @@ To add replicas to a DTR cluster, use the [join](/reference/dtr/2.7/cli/join/) c
     > Important
     >
     > The `<ucp-node-name>` following the `--ucp-node` flag is the target node on which to install the DTR replica. It is NOT the UCP Manager URL.
-    {: .important}
 
     When joining a replica to a DTR cluster, it is necessary to specify the
     ID of a replica that is already part of the cluster. Locate an
     existing replica ID on the **Shared Resources > Stacks** UCP page.
 
-3. Confirm that all replicas are running by navigating to UCP's web interface and selecting **Shared Resources > Stacks**. All replicas should display.
+3. Confirm that all replicas are running by navigating to the UCP web interface and selecting **Shared Resources > Stacks**. All replicas should be on display.
```

Comment:
I probably shouldn't give writing style feedback, because that's not my area of expertise, but I think 

All replicas should be displayed (sounds better?).

Reply:
or "You should see a list of all replicas."

Reply:
LOTS of input on this wee sentence (@traci-morrison actually chimed in first). Going this way:

"Confirm that all of the replicas are running by navigating to the UCP web interface and selecting **Shared Resources > Stacks** (a list of the running replicas should display)."